packages/tools/pie-icons-configs/__tests__/configs.test.js

```diff
@@ -110,126 +112,136 @@ describe('validateGetRegularIconSize', () => {
 });
 
 describe('getSvgProps', () => {
-    describe('when provided with a class for regular size icon', () => {
-        const iconName = 'IconAppOrder';
+    describe('when provided with a', () => {
+        describe('svgClasses parameter', () => {
+            it('returns an object that contains a class attr with the expected classes', () => {
+                const received = getSvgProps('icon-test', null, null);
```

Comment:
It looks like the `getSvgProps` function has 4 parameters but we're only calling it with 3? I might be missing something but I'd expect the linter to complain, at least

Reply:
The last parameter, `componentName`, is used for console error message so it is easier to understand where the error comes from. So I didn't pass it for tests that don't need it.

Reply:
I added it to all the tests for consistency